Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetLPMUITest.swift

```diff
@@ -149,6 +149,39 @@ class PaymentSheetStandardLPMUIOneTests: PaymentSheetStandardLPMUICase {
         XCTAssertTrue(app.staticTexts["Success!"].waitForExistence(timeout: 15.0))
     }
 
+        func testSunbitPaymentMethod() throws {
```

Comment:
Is this meant to be in this PR?

Reply:
Yeah, this was a follow up item from the Sunbit integration PR when @yuki-stripe noticed I was missing a test